apps/test/unit/templates/teacherDashboard/EmptySectionsTest.jsx

```diff
@@ -2,62 +2,82 @@ import {render, screen} from '@testing-library/react';
 import React from 'react';
 import {BrowserRouter as Router} from 'react-router-dom';
 
-import EmptySection from '@cdo/apps/templates/teacherDashboard/EmptySection';
+import EmptySection from '@cdo/apps/templates/teacherDashboard/EmptySectionV1';
```

Comment:
Yes, I noticed this in my next PR [here](https://github.com/code-dot-org/code-dot-org/pull/60451). This is still working because the `import NAME` will import the default from that file in whatever name we put (e.g. `NAME`). So this is still working, but we should definitely change it, could we do that in the next PR though?

Reply:
Sounds good to me!